### PR TITLE
issue templates created 

### PR DESCRIPTION
**Pull request by jvishnuvardhan**
issue templates created newly for keras-team/keras repo to categorize keras issues with different labels

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/00-bug-issue.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+---
+name: Bug Issue
+about: Use this template for reporting a bug
+labels: 'type:bug'
+
+---
+
+Please go to Stack Overflow for help and support:
```
Comment (fchollet) [inline .github/ISSUE_TEMPLATE/00-bug-issue.md:8]: Should we also advertise the TF forum? Should we *only* advertise the TF forum? What's our policy?

File: .github/ISSUE_TEMPLATE/30-feature-request.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+Are you willing to contribute it (Yes/No) :   
+
+**Describe the feature and the current behavior/state**.
+
+Describe the feature clearly here. Be sure to convey here why the requested feature is needed. Any brief description about the use-case would help.
+
+**Will this change the current api? How?**
+
+
+**Who will benefit with this feature?**
```
Comment (fchollet) [inline .github/ISSUE_TEMPLATE/30-feature-request.md:30]: benefit from

File: .github/ISSUE_TEMPLATE/10-build-installation-issue.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+---
+name: Build/Installation Issue
```
Comment (fchollet) [inline .github/ISSUE_TEMPLATE/10-build-installation-issue.md:2]: Keras does not have its own build process so we can remove this template. Build issues will always be routed to TF.

File: .github/ISSUE_TEMPLATE/20-performance-issue.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
+---
+name: Performance Issue
```
Comment (fchollet) [inline .github/ISSUE_TEMPLATE/20-performance-issue.md:2]: This template is virtually identical to the bug one so we can remove it. Users can file performance issues as bugs.

**Review by fchollet — COMMENTED** [review]
Thank you for the PR!

File: .github/ISSUE_TEMPLATE/00-bug-issue.md (hunk 1 above)
Comment (jvishnuvardhan) [reply .github/ISSUE_TEMPLATE/00-bug-issue.md:8]: For TF repositories, we are recommending TF forum only (no stackoverflow). Supporting TF Forums is easy and manageable. Please let me know whether to update it with TF Forum link.

File: .github/ISSUE_TEMPLATE/20-performance-issue.md (hunk 4 above)
Comment (jvishnuvardhan) [reply .github/ISSUE_TEMPLATE/20-performance-issue.md:2]: Sure.

**Review by fchollet — APPROVED** [review]
LGTM, thanks

**Review by gbaned — APPROVED** [review]

**Comment by fchollet** [comment]
I believe this PR should be merged directly rather than via Copybara.

**Comment by fchollet** [comment]
Please resolve the conflicts @jvishnuvardhan:

Conflicting files
.github/ISSUE_TEMPLATE/10-feature-request.md
.github/ISSUE_TEMPLATE/20-documentation-issue.md

**Comment by gbaned** [comment]
@jvishnuvardhan  Can you please resolve conflicts? Thanks!

**Comment by jvishnuvardhan** [comment]
I am closing this as this was done through a CL. Thanks!

**Pull request closed without merging**